modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoServerCodegen.java

```diff
@@ -106,6 +106,12 @@ public GoServerCodegen() {
         optAddResponseHeaders.defaultValue(addResponseHeaders.toString());
         cliOptions.add(optAddResponseHeaders);
 
+        // option to disallow unknown fields in body
```

Comment:
Suggestion: elaborating a bit more by providing an example on what this option does.

Reply:
And this option seems to be related to `disallowAdditionalPropertiesIfNotPresent`.

Can we reuse `disallowAdditionalPropertiesIfNotPresent` instead?

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConstants.java#L385-L388

Reply:
Thanks for the feedback. Reverted, and fixed the API.